src/app/src/components/AnimatedArcsAndMap/AnimatedTotalSpendingBucket.tsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from 'react';
+import { Progress, Tag, TagLabel } from '@chakra-ui/react';
+
+import { SpendingByGeographyAtMonth } from '../../types/api';
+import { TOTAL_BIL_AMOUNT } from '../../constants';
+
+export default function AnimatedTotalSpendingBucket({
+    spendingAtTimeByState,
+}: {
+    spendingAtTimeByState: SpendingByGeographyAtMonth;
+}) {
+    const [totalSpendingAtTime, setTotalSpendingAtTime] = useState(0);
+
+    useEffect(() => {
+        if (spendingAtTimeByState) {
+            const totalAwardsAtTime: number = Object.values(
+                spendingAtTimeByState
+            ).reduce((sum, stateResults) => {
+                sum += stateResults && stateResults.aggregated_amount;
+                return sum;
+            }, 0);
+            totalAwardsAtTime && setTotalSpendingAtTime(totalAwardsAtTime);
+        }
+    }, [spendingAtTimeByState]);
+
+    const amountLeft = divideByBillion(TOTAL_BIL_AMOUNT - totalSpendingAtTime);
+
+    return (
+        <Tag
+            background='none'
+            className={'leaflet-bottom'}
+            position={'absolute'}
+            right={'60'}
+        >
+            <Progress
+                value={amountLeft}
+                opacity={100}
+                colorScheme={'progress'}
+                aria-label='spending-progress-bar'
+                height={'20px'}
+                width={'60px'}
+                style={{ transform: 'rotate(270deg)' }}
+                min={0}
+                max={550}
```

Comment:
```suggestion
                max={divideByBillion(TOTAL_BIL_AMOUNT)}
```

Or maybe leave it as billions and do the conversion just for the label.